pom.xml

```diff
@@ -5,7 +5,7 @@
 
     <groupId>org.nem.nis</groupId>
     <artifactId>nem-nis</artifactId>
-    <version>0.6.98</version>
+    <version>0.6.99</version>
```

Comment:
i guess this was missed as part of last version bump ?

Reply:
🤔, possibly